### ISSUE TITLE
Directory name gets duplicated and combined

### ISSUE THREAD

**Issue by NeptuniusDexter** [dus4nstojanovic/create-files-from-template-vscode#1]
When trying to use the templates, it takes the directory of the project and appends it onto the directory of the project creating an invalid path. 

For example lets say I have a project in the C:\ root. If I then try and use my template in the C:\ root, I will get this error:
Couldn't determine if the provided path is a directory: 'c:\c:\template'

So it like doubles the root directory. I attached a screenshot of the error I received:

![image](https://github.com/user-attachments/assets/34df1a63-fafe-4684-a0f8-0f812f826976)

**Comment by dus4nstojanovic** [comment]
@NeptuniusDexter Thank you for reporting. Can you please share your template configuration? I will take a look.

**Comment by NeptuniusDexter** [comment]
This is my cfft.config.json. It is the default one that gets generated. The templates I am using are also the default which get generated.

### cfft.config.json
`{
  "defaultTemplateName": "component",
  "templates": [
    {
      "name": "component",
      "options": {
        "templatePath": "/.cfft.templates/component",
        "dirPath": "./{fileName}",
        "fileNameTextToBeReplaced": "component",
        "searchAndReplace": [
          {
            "search": "FileName",
            "replace": "{fileName}"
          }
        ]
      }
    }
  ]
}`

# .cfft.templates
## component
### component.module.scss
`.root {
  display: block;
}
`

### component.tsx
```
import { FC } from "react";
import styles from "./FileName.module.scss";

const FileName: FC = () => {
  return <div className={styles.root}>FileName</div>;
};

export default FileName;
```

**Comment by dus4nstojanovic** [comment]
@NeptuniusDexter I identified the issue. There is a problem with the `templatePath` for Windows users.

**Comment by NeptuniusDexter** [comment]
It seems to be working now. How would I configure it not to generate a
folder each time? I made my own template and put it directly in the root of
.cfft.templates (not under a component file as with the default templates).
I also then changed the "templatePath" to be equal to:  "templatePath":
"./.cfft.templates", yet when I generate a file from the template it still
gets put inside of a folder.

**Comment by dus4nstojanovic** [comment]
Actually, it doesn't work correctly if you try to create files in any inner folder. 

It turned out that the logic for generating paths is wrong for all Windows users.
I plan to invest some time to fix it this weekend.

I will let you know and help you with the configuration once I finish. 

Thank you for understanding 🙂

**Comment by NeptuniusDexter** [comment]
Thanks, appreciate it

**Comment by dus4nstojanovic** [comment]
@NeptuniusDexter The issue has been fixed in the latest extension version: `1.6.6`. Please update the extension and try it out.

Regarding your question on how to set it up to generate a file without a folder, check this link: https://marketplace.visualstudio.com/items?itemName=DusanStojanovic.cfft#creating-a-single-file

Example:
1. Create a single template file: `/.cfft.templates/yourFile.ts`.
2. Add the template configuration in the templates array in the `.cfft.confing.json`:
```json
{
  "name": "your-template-name",
  "options": {
    "templatePath": "/.cfft.templates/yourFile.ts",
    "dirPath": ".",
    "fileNameTextToBeReplaced": "yourFile",
    "shouldReplaceFileContent": false
}
```

- `templatePath: "/.cfft.templates/yourFile.ts"` leads to the template file
- `dirPath: "."` tells the app that you want to create a file in the selected folder
- `fileNameTextToBeReplaced: "yourFile"` - you want to replace the `yourFile` part of the template file name with the file name you provide,
- `shouldReplaceFileContent: false` - means that the file content will stay unchanged.

> However, if you want to also replace some parts of the file content, you can add the `searchAndReplace ` or `textToBeReplaced` and `replaceTextWith` fields.

Please let me know if everything worked as expected so that I can close the issue 🙂

**Comment by NeptuniusDexter** [comment]
Thank you,

It is working now. Appreciate the assistance.
